packages/create/src/cli/change-data-source/change-data-source.ts

```diff
@@ -0,0 +1,106 @@
+import type { Command } from 'commander';
+import type { AdpProjectData, DataSourceData, PromptDefaults } from '@sap-ux/adp-tooling';
+import {
+    generateChange,
+    ChangeType,
+    ChangeDataSourcePrompts,
+    getTargetDataSources,
+    getTargetODataAnnotations,
+    getDataSourcesDictionary,
+    getDataServicesWithURI
+} from '@sap-ux/adp-tooling';
+import { prompt } from 'inquirer';
+import { createForAbap } from '../../../../axios-extension';
+import { readFileSync } from 'fs';
+import { join } from 'path';
+import { UI5Config } from '@sap-ux/ui5-config';
+
+/**
+ * Add a new sub-command to change the data source of an adaptation project to the given command.
+ *
+ * @param {Command} cmd - The command to add the change-data-source sub-command to.
+ */
+export function addChangeDataSourceCommand(cmd: Command): void {
+    cmd.command('change-data-source')
+        .argument('<path>', 'path to the Adaptation Project folder')
+        .requiredOption('--url [url]', 'url pointing to the target system containing the original app')
+        .requiredOption('--username [username]', 'username for the target system containing the original app')
+        .requiredOption('--password [password]', 'password for the target system containing the original app')
+        .requiredOption('--reference [reference]', 'id of the original application')
+        .action(async (path, options) => {
+            await changeDataSource(path, { ...options });
+        });
+}
+
+/**
+ * Changes the data source of an adaptation project.
+ *
+ * @param {string} basePath - The path to the adaptation project.
+ * @param {PromptDefaults} defaults - The default values for the prompts.
+ */
+async function changeDataSource(basePath: string, defaults: PromptDefaults): Promise<void> {
+    if (defaults.url && defaults.username && defaults.password && defaults.reference) {
+        const url = new URL(defaults.url);
+        const provider = createForAbap({
+            baseURL: url.origin,
+            ignoreCertErrors: true,
```

Comment:
no, sorry, you need to ask the user or allow to provide it as CLI parameter, but you cannot disable SSL validation by default.